.env.example

```diff
@@ -3,4 +3,7 @@ SENTRY_PROJECT=
 SENTRY_AUTH_TOKEN=
 SENTRY_DSN=
 DOMAIN_FORMAT=
-
+APP_BUILD_VERSION=1
+APP_BINARY_VERSION=dev
+PROJECT_ID=
```

Comment:
needed for ci, but our app config is set in a way to fail if env vars arent defined